python/lsst/ip/isr/isrTask.py

```diff
@@ -439,7 +439,8 @@ def readIsrData(self, dataRef, rawExposure):
     def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, defects=None,
             fringes=None, bfKernel=None, camera=None,
             opticsTransmission=None, filterTransmission=None,
-            sensorTransmission=None, atmosphereTransmission=None):
+            sensorTransmission=None, atmosphereTransmission=None,
+            otherDataRef=None):
```

Comment:
The way you've implemented the decam crosstalk correction, I don't think that modifications here are necessary.  I think this should be reverted except to add a `prepCrosstalk` method to the `CrosstalkTask` base class.

Reply:
That method would be called in the `RunDataRef` method on IsrTask.  You would still have to pass through the source data object, but then it wouldn't be opaque.